package.json

```diff
@@ -121,6 +121,7 @@
     "react": "^16.3.2",
     "react-dom": "^16.3.2",
     "react-styleguidist": "^8.0.6",
+    "react-transition-group": "^2.2.1",
```

Comment:
This package be added to the dependencies of SideNavigation.

Reply:
react-transition-group is used in the /shared directory and not in SideNavigation....

Reply:
Removed from package.json